doc/classes/VisualShader.xml

```diff
@@ -193,7 +193,6 @@
 		</method>
 	</methods>
 	<members>
-		<member name="code" type="String" setter="set_code" getter="get_code" override="true" default="&quot;shader_type spatial;void vertex() {// Output:0}void fragment() {// Output:0}void light() {// Output:0}&quot;" />
```

Comment:
CC @Chaosus.

This override with VisualShader's default code was correct, but a bit confusing IMO, so I put an `ADD_PROPERTY_DEFAULT` to hide it.

But if you think that it's better to have it explicitly here, we can keep it. In the current `master` branch it would then change to:
```
&quot;shader_type spatial;render_mode specular_schlick_ggx;void vertex() {// Output:0}void fragment() {// Output:0}void light() {// Output:0}&quot;
```

Reply:
It seems redundant for other users, so ok